libs/liblobby/lobby/lobby.lua

```diff
@@ -898,6 +898,11 @@ end
 -- Bots have isBot=true, AIs have aiLib~=nil and humans are the remaining
```

Comment:
Docs above this line don't even mention `status.owner`. Is this a key that exists? If so, docs should be updated - or maybe removed if they're confusing.

That said, I can see `status.owner` being used below so it probably does exist and I can understand if you don't want to do it now - it's not a part of your PR technically.